runtime/Tracer.h

```diff
@@ -0,0 +1,58 @@
+
+#ifndef SYMRUNTIME_TRACER_H
+#define SYMRUNTIME_TRACER_H
+
+#include "GarbageCollection.h"
+#include "Runtime.h"
+#include <cstdint>
+#include <fstream>
+#include <nlohmann/json.hpp>
```

Comment:
Json library has not been added to SymCC dependencies. Please update the README, dockerfile and github workflows.